app/views/shared/_links_table.html.erb

```diff
@@ -0,0 +1,55 @@
+<%
+  @table_rows = links.map do |link|
+    si = link.service_interaction
+    title = si.govuk_title || link.service.label
+    pres = LinkPresenter.new(link)
+    [
+      { text: link.analytics.to_i, format: "numeric" },
+      { text: "<span class=\"app-service-label\">#{title}</span> <span class=\"app-interaction-label\">(#{link.interaction.label})</span>".html_safe },
+      { text: link_to(link.local_authority.name, local_authority_path(link.local_authority), class: "govuk-link") },
+      { text: "<span class=\"govuk-tag govuk-tag--#{pres.status_tag_colour}\">#{pres.status_description}</span>".html_safe },
+      { text: pres.last_checked },
+      { text: link_to("Edit <span class=\"govuk-visually-hidden\">#{link.local_authority.name} - #{si.service.label} - #{si.interaction.label}</span>".html_safe, edit_link_path(link.local_authority, link.service, link.interaction), class: "govuk-link") },
+    ]
+  end
+
+  @headers = [
+    {
+      text: "Visits this week",
+      format: "numeric",
+    },
+    {
+      text: "Service",
+    },
+    {
+      text: "Council",
+    },
+    {
+      text: "Status",
+    },
+    {
+      text: "Last Checked",
+    },
```

Comment:
I'm wondering if we can remove the 'last checked' column on the table page. The information always seems to read 'last checked one year ago' or 'not checked'. The same information is available on the 'show' page. Removing it would allow for more space in the table.

Reply:
Hmm. If you look at it on production or integration it's more like "8 hours ago" etc. I'll agree it's probably not that useful though. We can always put it back if necessary.